### PR TITLE
docs: Fix some typo

### PR DESCRIPTION
**Pull request by iliul**

### CODE REVIEW

**Comment by johnzeng** [comment]
Thanks for fixing typo, but I can't accept this because of the renaming. The file name must strictly match the original file name on `https://rocksdb.org.cn/doc.html` so they can sync files. Please change them back and then I can merge this PR

**Comment by iliul** [comment]
> The file name must strictly match the original file name on `https://rocksdb.org.cn/doc.html` so they can sync files. 

Agree, will revert this later.

> Please change them back and then I can merge this PR

Please keep this open until i remove `DNM` prefix in pr title :smile:

File: doc/RocksDB-FAQ.md [hunk 1]
```diff
@@ -153,7 +153,7 @@
 **问:一个迭代器需要持有多少资源，这些资源会在什么时候被释放？**
 
 答：迭代器需要持有数据块以及内存的memtable。每个迭代器需要持有以下资源：
-- 当前的迭代器所指向的所有数据块。参考 [迭代器固定的数据块]()
+- 当前的迭代器所指向的所有数据块。参考 [迭代器固定的数据块](Memory-usage-in-RocksDB.md#blocks-pinned-by-iterators)
```
Comment (johnzeng) [inline doc/RocksDB-FAQ.md]: 这里子标题不对，应该是'迭代器固定的块'，因为我把子标题也翻译了

**Comment by johnzeng** [comment]
Any update?

File: doc/RocksDB-FAQ.md (hunk 1 above)
Comment (iliul) [reply doc/RocksDB-FAQ.md]: > 这里子标题不对，应该是'迭代器固定的块'，因为我把子标题也翻译了

Fixed

**Comment by iliul** [comment]
> Any update?

链接修复了

最近两天忙，过几天继续学习，感谢你翻译的文档，很有用~

**Comment by johnzeng** [comment]
是否需要给你署名？可以加入到校订者名单哈。

**Comment by iliul** [comment]
> 是否需要给你署名？可以加入到校订者名单哈。

不用，贡献多了后再考虑吧！

**Pull request merged**